patch_api/api/estimates_api.py

```diff
@@ -18,7 +18,10 @@
 # python 2 and python 3 compatibility library
 import six
 
-from patch_api.exceptions import ApiTypeError, ApiValueError
+from patch_api.exceptions import (
+    ApiTypeError,
+    ApiValueError
+)
```

Comment:
With python, since the beginning, I have been getting a different formatting than Lovisa had.

I'm not completely sure which one is the expected one given the settings. If mine, this might be worth updating once and for all.